Automattic/wp-calypso
# Domain Forwarding: Supporting advanced settings

Related to 3534-gh-Automattic/dotcom-forge

| Desktop  | Mobile |
| ------------- | ------------- |
| <img width="760" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/55b47f26-c27c-4dc8-b240-18c959ace690">  | <img width="484" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/736e86bc-d9ca-44c1-966b-09f9e6c67f2c">  |


## Proposed Changes

* Added Advanced support to Redirect type and Path forwarding on Domain Forwarding feature
* Delete button now cleans the input (thanks for your suggestion @lsl)

## Testing Instructions



* Apply this diff into your sandbox first: D119630-code
* Open Domain Forward feature and validate both Redirect type and Path Forwarding are working as expected
* Ensure we only release save button if a change on frontend data happened
* Ensure deleting and adding new Domain Forward will work as expected
* Try to break it 😢 


## Pre-merge Checklist
